src/spriteSheet.js

```diff
@@ -89,14 +91,16 @@
       // get the row and col of the frame
       let row = this.frames[this._f] / this.spriteSheet._f | 0;
       let col = this.frames[this._f] % this.spriteSheet._f | 0;
-
-      (properties.context || kontra.context).drawImage(
+      let width = (properties.width !== undefined) ? properties.width : this.spriteSheet.frame.width
```

Comment:
Same as the image logic, would probably be safe to assume `0` wouldn't be passed, so could shorten to `properties.width || this.width`. Also, the animations width and height are set to `this.spriteSheet.frame.width/height`, so there shouldn't be a need to expand it from `this.width` and `this.height`.